lib/plugins/aws/deploy/compile/functions/index.js

```diff
@@ -183,6 +176,40 @@ class AwsCompileFunctions {
 
       _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Outputs,
         newOutputObject);
+
+      if (typeof this.serverless.service.provider.iamRoleARN !== 'string') { // only if default IAM
+        const logGroupTemplate = `
+          {
+            "${normalizedFunctionName}LogGroup": {
+              "Type" : "AWS::Logs::LogGroup",
+              "Properties" : {
+                "LogGroupName" : "/aws/lambda/${FunctionName}"
+              }
+            }
+          }
+        `;
+        const newLogGroup = JSON.parse(logGroupTemplate);
+        _.merge(this.serverless.service.provider.compiledCloudFormationTemplate.Resources,
+          newLogGroup);
+
+        this.serverless.service.provider.compiledCloudFormationTemplate
+            .Resources
+            .IamPolicyLambdaExecution
+            .Properties
+            .PolicyDocument
+            .Statement[0]
+            .Resource
+            .push({
+              'Fn:Join': [
```

Comment:
Sorry, this is exactly the problem with developing when not directly verifying.

This is changed in code and tests.